src/PerfView/PerfViewData.cs

```diff
@@ -5804,7 +5823,7 @@ protected override Action<Action> OpenImpl(Window parentWindow, StatusBar worker
                         hasMemAllocStacks = true;
                     if (name.StartsWith("GC/SampledObjectAllocation"))
                         hasMemAllocStacks = true;
-                    if (name.StartsWith("GC/CCWRefCountChange"))
+                    if (name.StartsWith("GC/CCWRefCountChange") || name.StartsWith("TaskCCWRef"))
```

Comment:
I want this view to be independent of the existing CCW refcount view (because it is only for .NET Native but the other is useful for .NET Desktop).

Reply:
done